near-sdk/src/collections/vector.rs

```diff
@@ -210,41 +244,31 @@ where
         let raw_element = Self::serialize_element(element);
         Self::deserialize_element(&self.replace_raw(index, &raw_element))
     }
+}
 
-    /// Removes all elements from the collection.
-    pub fn clear(&mut self) {
-        for i in 0..self.len {
-            let lookup_key = self.index_to_lookup_key(i);
-            env::storage_remove(&lookup_key);
-        }
-        self.len = 0;
-    }
-
-    /// Iterate over deserialized elements.
-    pub fn iter<'a>(&'a self) -> impl Iterator<Item = T> + 'a {
-        self.iter_raw().map(|raw_element| Self::deserialize_element(&raw_element))
-    }
-
-    /// Extends vector from the given collection.
-    pub fn extend<IT: IntoIterator<Item = T>>(&mut self, iter: IT) {
-        for el in iter {
-            self.push(&el)
-        }
+impl<T: fmt::Debug + BorshDeserialize> fmt::Debug for Vector<T> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        self.to_vec().fmt(f)
```

Comment:
this could potentially be expensive. If a user log this causally in the code, it might consume a lot of gas.

Reply:
I know. What do you suggest? I don’t think it should be used in production code, so it should be fine

Reply:
Not sure. @evgenykuzyakov what do you think?